features/environment.py

```diff
@@ -202,38 +207,70 @@ def create_uat_lxd_image(context: Context, series: str) -> None:
         )
         return
     now = datetime.datetime.now()
+    ubuntu_series = "ubuntu-daily:%s" % series
+
+    #if we are building the package from PR, we need a 2nd_base_image
+    #without the durty build dependencies from that 1st_base_image
```

Comment:
Thanks Lucas. Fixed it. That's the ideia: run automatically on travis-CI
I'm editing the comment - adding the related issues : 853 and 857

Reply:
Sorry, I think I have not described my idea properly. It would be to facilitate the process of running the behave test on the local machine, like you are creating a new behave tests and wants to see if it is working fine before opening the PR. I know your code already does that, but I was thinking on creating a script that the user will just run and it will run all of the behave tests for him, without worrying about creating a lxc container or creating the tarball with the current version of the code. But this is just an idea, I don't have any implementations idea for it yet